src/openqaoa-core/utilities.py

```diff
@@ -1427,6 +1484,76 @@ def flip_counts(counts_dictionary: dict) -> dict:
     return output_counts_dictionary
 
 
+def permute_counts_dictionary(
```

Comment:
This is function is already part of `dev`. Strange that it gets detected as new function. Need to check this